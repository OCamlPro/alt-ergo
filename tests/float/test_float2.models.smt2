(set-logic ALL)

(set-option :produce-models true)
(declare-const x Real)
(assert (> x 0.0))
(assert (= ((_ ae.round 1 2 RTZ) 0.3) x))
(check-sat)
(get-model)