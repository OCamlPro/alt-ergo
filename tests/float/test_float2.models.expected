
unknown
(
  (define-fun x () Real (/ 1 4))
)
